ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/fulu/helpers/MiscHelpersFulu.java

```diff
@@ -340,7 +364,11 @@ public List<DataColumnSidecar> reconstructAllDataColumnSidecars(
         existingSidecars.stream().findFirst().orElseThrow();
```

Comment:
Not part of this PR directly but I am curious why do we throw when we do not have any sidecars? Wouldn't it make more sense if we only returned an empty list in this case?

Reply:
We have assertion on the list size on the first line of this method. If we want to support both 0 and 64+ inputs we need to modify this assertion.